reference-implementation/to-upstream-wpts/resources/test-utils.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+self.getterRejects = (t, obj, getterName, target) => {
```

Comment:
This whole file already exists upstream; I just am copying it here because of limitations of the test runner.